include/engine/plugins/plugin_base.hpp

```diff
@@ -237,7 +237,7 @@ class BasePlugin
             phantom_nodes[i] = facade.NearestPhantomNodes(
                 parameters.coordinates[i],
                 number_of_results,
-                use_radiuses ? parameters.radiuses[i] : boost::none,
+                use_radiuses ? parameters.radiuses[i] : default_bearing_radius,
```

Comment:
May be I don't get something, but by current semantics it is just `default_radius`, but not `default_bearing_radius`, right? I.e. if it is passed we always use it as default radius, isn't it? I don't mind about having such behaviour, but lets rename it to just `default_radius` may be then?

Reply:
Oh, that's right. I could alternatively make a few adjustments to only apply the passed `default_bearing_radius` if bearings are used in the query - would that be preferable, as it's more inline with the original issue?

Reply:
In this case, should this function be affected by this new default_radius behavior?

https://github.com/Project-OSRM/osrm-backend/blob/192d077adad5bc0d1a5fb9d185c5c4113deabacb/include/engine/plugins/plugin_base.hpp#L166-L171

I've left it alone since it has it's own radius behavior (even for queries not containing bearings), but I'm not sure if this is something the modified use case should affect or not.